docs/style.css

```diff
@@ -393,3 +395,15 @@ body.dark-mode #abandon-game:active {
     background-color: #FFEB3B;
     border-color: #e57373;
 }
+
+#viewport{
+    padding: none;
+    display: block;
+    position: absolute;
+    box-sizing: border-box;
+}
+.canvas-container{
+    width: 850px;
+    height: calc(100vh - 60px);
+    padding-bottom: 0px !important;
```

Comment:
hmmm.... is there a way to avoid this `!important`?

Pro-tip: more specific css rules take priority over less specific ones. So something like `div.canvas-container` would take priority over `div` or `.canvas-container`.

I'm not 100% on the details, but I think that ids (`#id`) take priority over classes (`.class`), which take priority over element types. (`div`)